jupyterhub/handlers/pages.py

```diff
@@ -98,19 +95,24 @@ class SpawnHandler(BaseHandler):
             if not user.admin:
                 raise web.HTTPError(403, "Only admins can spawn on behalf of other users")
 
+
             user = self.find_user(for_user)
             if user is None:
                 raise web.HTTPError(404, "No such user: %s" % for_user)
 
+
```

Comment:
Just curious: is there a reason that all these extra blank lines were added?

Reply:
I recall trying to follow the formatting practice be consistent with what I saw within the file. I dislike that it added a complexity to the PR so I will remove these changes to keep the PR very focused! I should practice on this.